### DISCUSSION TITLE
React native storybook in a confusing state

### DISCUSSION THREAD

**Discussion by dannyhw** [storybookjs/storybook#11762] [Help] [answered]
Hey guys, first of all I want to say great job on an awesome project and thanks for all your hard work! I think storybook is awesome and I want to start with that.

I wasn't sure about where to put this so sorry if I'm posting this in the wrong place etc.

So I'm currently working on a react native project where we are using the storybook for react native and have experienced some issues. Specifically since recent updates to react native there are some warnings and crashes that happen. When I started trying to investigate the problem and see if anyone else has reported the issues is where the confusion started. 

The first thing that is confusing is that there are two repositories with react native storybook code:
1. here in this repo 
2. at https://github.com/storybookjs/react-native.

However the storybookjs/react-native repo has no README.md and has been inactive since may.

I actually reached out to gongreg as he seemed to be the main contributor to the react native storybook. He was kind enough to respond and share some information about what's happened. 

From what he told me I understand that the react native code was in the process of being migrated to its own mono repo which is the storybookjs/react-native repo. However gongreg has been taking a break from maintaining the project which meant he was unable to finish the work on project to get it ready for contributors.

To clarify I think it's perfectly fine for him to take a break and I'm only mentioning it here because I want to see if there is a way someone else can help out and look at PR's there whilst he is not available.

I would like to try and help out by contributing my own PR's to the native project, however I get the impression that the PR's wouldn't be seen and also I have a hard time understanding how to run the project, where the tests run from and these sorts of things since the readme is not completed. I don't want to hassle gongreg as I think it's within his rights to take a break so I was wondering if there is anyone else with the permissions on the project that would be able to look into it.
For example it would be great if there was some details on how to contribute and someone who can look at PR's and issues.

Hopefully I haven't made incorrect assumptions here or understood the situation incorrectly, please let me know if so.

Also if I can help out in any way I'm willing to try 🙂

Thanks, Daniel.

**Comment by shilman** [comment]
Since you've already been in touch with gongreg you probably know more than I do.

Regarding your question about code, there is no longer any RN code in the monorepo's `next` branch, where we do our main development.

Regarding maintenance of the RN repo, I believe @ndelangen was helping gongreg get the repo ready. Perhaps he can help you get up to speed there if you're interested in maintaining it? It would be great to see the project move forward.

**Reply by dannyhw** [reply]
Hey, 

Yeah sure I'd be willing to maintain it the best I can :)

**Comment by ndelangen** [comment, marked as answer]
Hello @dannyhw 

Let's schedule a meeting!
https://calendly.com/ndelangen/storybook

Perhaps we could get @Gongreg on board in that meeting as well? 🙏

**Reply by dannyhw** [reply]
Ok sure thanks @ndelangen I appreciate it :).

**Reply by dannyhw** [reply]
Did you get my email/meeting details?
